src/tag.tsx

```diff
@@ -33,7 +34,11 @@ const Tag: React.FC<TagProps> = ({Icon, children, dataAttributes, type = 'promo'
     const tagTypeToColors = {
         promo: [colors.promoHigh, colors.promoHighInverse, colors.promoLow],
```

Comment:
side-side note: I think there's room for more semantics some way: let me explain this better.
I've had to expand the sourcecode in order to understand what position was for (the `[textColor, inverseTextColor, backgroundColor]` below). Although it's not such a big problem, I'm wondering if there's a better way to make this variable more semantic (i.e. an object with properties) where it's defined, and not where it's used 🙃

Reply:
yes, we could have that, but the resulting code would be quite bigger because we have to repeat the same object keys for every tag.
Maybe a comment above would be enough, wdyt?